src/PoolManager.sol

```diff
@@ -277,7 +277,7 @@ contract PoolManager is IPoolManager, ProtocolFees, NoDelegateCall, ERC6909Claim
     }
 
     /// @inheritdoc IPoolManager
-    function sync(Currency currency) external {
+    function sync(Currency currency) external onlyWhenUnlocked {
```

Comment:
what is the reason for this change?

Reply:
It's unnecessary for the manager to be unlocked to `sync` before and after this change.